cfpb/cfgov-refresh
# Updates Chart footer with cleared language

Changes "each visualization" to "this visualization".

## Testing

1. Load your local dev server
1. Open a page with a Chart on it, e.g. http://localhost:8000/data-research/consumer-credit-trends/auto-loans/origination-activity/
1. Verify that the chart footers now say "this visualization" instead of "each visualization"

## Checklist
